### ISSUE TITLE
Version 2.1.1 Does not compile

### ISSUE THREAD

**Issue by bimalsubedi** [sdl/dxa-web-application-java#118]
Hello,
We are using the dxa version of 2.1.1 that was uploaded to maven repository in March  in our web application however it seems that there is dependency problem with the library **udp-api-broker**. I am getting the following error **Could not resolve dependencies for project Failed to collect dependencies at com.sdl.dxa:dxa-tridion-provider:jar:2.1.1 -> com.sdl.dxa:dxa-tridion-common:jar:2.1.1 -> com.sdl.delivery:udp-api-broker:jar:(,11.0.1): No versions available for com.sdl.delivery:udp-api-broker:jar:(,11.0.1) within specified range -> [Help 1]**
I also cloned the source code of release/2.1.1 and it also gives the same error. Is there any prevision to resolve this error?

**Comment by rpannekoek** [comment]
There appears to be a mistake here; we did not release DXA 2.1 Java at all.
We will do a DXA Model Service 2.1.1 hotfix release (DXA 2.1 .NET release does contain the DXA Model Service, which is implemented in Java),  but the other DXA Java artifacts should not have been released to Maven Central.

**Comment by bimalsubedi** [comment]
So is it possible to remove that version from maven repository and leave only the stable version? This brings us a lot of confusion and now we don't know which is the latest stable version of DXA that we can use. Would appreciate if you could provide me with that information.

**Comment by rpannekoek** [comment]
Yes, we will work on cleaning this up to avoid the confusion.
For clarity: for DXA Java, version 2.0 is the latest release (you can use the `release/2.0` branch to obtain the sources).
Note that since the 2.0 release, several hotfix releases have been done (focused at the DXA Model Service), so you will find 2.0.2 artifact versions.  The `release/2.0` branch includes those hotfixes (you can use the `DXA_2.0_Release` tag if you want to get the sources of DXA 2.0.0 (i.e. excluding hotfixes).

**Comment by fdvalencia** [comment]
Hi Rick I have tried to use version 2.0.2 version but its missing a dependency:

dxa-module-core 2.0.2

its not uploaded to the mvn repository

**Comment by rpannekoek** [comment]
This is very unfortunate...
We recently did a DXA 2.0.2 Model Service hotfix release, but a blocking issue has been found after release, which required us to release 2.0.3 versions of the DXA framework.
Indeed, not all 2.0.2 artifacts were published yet; we will work on publishing all 2.0.2 artifacts to make things a bit less confusing.
But note that you should not use the 2.0.2 artifacts, but use the 2.0.3 ones instead.  By now, the `release/2.0` branch also references these 2.0.3 versions.

**Comment by rpannekoek** [comment]
Internal Issue ID for cleaning up the confusion around DXA 2.1 release: TSI-3900

**Comment by rpannekoek** [comment]
Unfortunately, it is not possible to unpublish the DXA 2.1.1 artifacts from Maven Central, however we have cleaned up GitHub, so there are no references to DXA 2.1.x artifacts anymore.
In particular:

- The {{release/2.1}} branch has been removed
- The {{develop}} and {{master}} branches currently represents DXA 2.2 development (in progress) and thus references {{2.2.0-SNAPSHOT}} versions.

I hope this is sufficient to avoid confusion.
